recipes/fixtures/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{%set name = "fixtures" %}
+{%set version = "3.0.0" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "fcf0d60234f1544da717a9738325812de1f42c2fa085e2d9252d8fff5712b2ef" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pbr >=0.11
+
+  run:
+    - python
+    - extras
+    - pbr >=0.11
+    - six
+    - testtools >=0.9.22
+
+test:
+  imports:
+    - fixtures
+    - fixtures.callmany
+    - fixtures.fixture
+    - fixtures.testcase
+
+
+about:
+  home: https://github.com/testing-cabal/fixtures
+  license: Apache 2.0 or BSD 3-Clause
+  license_file: COPYING
```

Comment:
Sort of. Seems to include a reference to both without the licenses per se.

That being said, this case feels like a `conda-build` issue IMHO as there is not a manner of specifying two licenses for cases just like this. Please see issue ( https://github.com/conda/conda-build/issues/1247 ).